core/consensus/aura/src/lib.rs

```diff
@@ -437,7 +437,9 @@ impl<B: Block, C, E, I, P, Error, SO> SlotWorker<B> for AuraWorker<C, E, I, P, S
 }
 
 /// check a header has been signed by the right key. If the slot is too far in the future, an error will be returned.
-/// if it's successful, returns the pre-header, the slot number, and the signat.
+/// if it's successful, returns the pre-header and the digest item containing the seal.
+///
+/// This digest item will always return `Some` when used with `as_aura_seal`.
```

Comment:
It would be nice to enforce this at the type level, but that should not block merging.